src/ServicePulse.Host/app/js/views/archive/view.html

```diff
@@ -51,7 +51,7 @@
                     </div>
 
                     <div class="col-xs-11 failed-message-data" ng-mouseenter="message.hover2 = true" ng-mouseleave="message.hover2 = false">
-                        <a href="#/failedMessages/{{message.id}}">
+                        <a href="#/message/{{message.id}}">
```

Comment:
> Either way, I would tackle this as a separate issue, simply becasue group id path is already out there.

Yeah, thought about that as soon as I started seeing you disagree :-)

Let's leave URL formatting out of the picture for now and release the rest.